setup.py

```diff
@@ -19,8 +19,22 @@
 from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 from version import version
+from pkg_resources import parse_version
 
-toil_version = '3.1.4'
+# Toil version check -- Raise warning instead of using intall_requires to avoid virtualenv conflicts
+toil_min_version = '3.1'
```

Comment:
I think you want to mention a concrete version here, 3.1.6 to be specific. There is no reason to allow, say 3.1.4 because 3.1.6 is fully backwards-compatible with 3.1.4 and the upgrade should be straightforward.

That reminds me, I should bump Toil on toil-jenkins-slave in cgcloud.